packs/feats/fiend-eternal.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p>Through all your machinations and accumulation of power, you have completed your apotheosis and become a true archfiend. You gain the fiend and unholy traits and are immune to your realm's damage type. You don't age. Your planned realm becomes a complete reality, forming within a section of Hell, the Outer Rifts, the Netherworld, its own fiendish demiplane, or some other planar space of your choosing. If you have the @UUID[Compendium.pf2e.feats-srd.Item.Seat of Power] feat, that seat of power is grafted onto your realm and the @UUID[Compendium.pf2e.actionspf2e.Item.Enter Seat Of Power] activity transports you to any place within your realm; otherwise, you can enter and exit your realm in a similar fashion with a brief ritual that takes 10 minutes (though you can't bring anyone with you).</p>\n<p>Whenever you would die, you are instead reduced to 0 Hit Points, become @UUID[Compendium.pf2e.conditionitems.Item.Unconscious], and are immediately transported to your seat of power within your realm. After 1 hour (or 1 round if you have the Seat of Power feat), you regain @Damage[5[healing]]{5 Hit Points} and reawaken. You can't leave your realm until you're fully healed or 1 week passes (whichever comes first).</p>\n<p>In addition, you gain a small group of fiendish followers who reside in your realm. They never leave your realm (though one might occasionally come to your aid if you have the @UUID[Compendium.pf2e.feats-srd.Item.Marshall Fiendish Forces] feat), but they constantly spy on the mortal world to keep you informed. You are telepathically linked to these followers and can choose to speak with the group, or a specific fiend, as a three-action activity that has the concentrate trait. In doing so, you can avail yourself of the fiends' surveillance to learn about the world or current events. You can either Recall Knowledge on any topic or @UUID[Compendium.pf2e.actionspf2e.Item.Gather Information]. If you critically fail to do either, you get a failure instead.</p>"
```

Comment:
```suggestion
            "value": "<p>Through all your machinations and accumulation of power, you have completed your apotheosis and become a true archfiend. You gain the fiend and unholy traits and are immune to your realm's damage type. You don't age. Your planned realm becomes a complete reality, forming within a section of Hell, the Outer Rifts, the Netherworld, its own fiendish demiplane, or some other planar space of your choosing. If you have the @UUID[Compendium.pf2e.feats-srd.Item.Seat of Power] feat, that seat of power is grafted onto your realm and the @UUID[Compendium.pf2e.actionspf2e.Item.Enter Seat Of Power] activity transports you to any place within your realm; otherwise, you can enter and exit your realm in a similar fashion with a brief ritual that takes 10 minutes (though you can't bring anyone with you).</p>\n<p>Whenever you would die, you are instead reduced to 0 Hit Points, become @UUID[Compendium.pf2e.conditionitems.Item.Unconscious], and are immediately transported to your seat of power within your realm. After 1 hour (or 1 round if you have the Seat of Power feat), you regain @Damage[5[healing]]{5 Hit Points} and reawaken. You can't leave your realm until you're fully healed or 1 week passes (whichever comes first).</p>\n<p>In addition, you gain a small group of fiendish followers who reside in your realm. They never leave your realm (though one might occasionally come to your aid if you have the @UUID[Compendium.pf2e.feats-srd.Item.Marshall Fiendish Forces] feat), but they constantly spy on the mortal world to keep you informed. You are telepathically linked to these followers and can choose to speak with the group, or a specific fiend, as a three-action activity that has the concentrate trait. In doing so, you can avail yourself of the fiends' surveillance to learn about the world or current events. You can either @UUID[Compendium.pf2e.actionspf2e.Item.Recall Knowledge] on any topic or @UUID[Compendium.pf2e.actionspf2e.Item.Gather Information]. If you critically fail to do either, you get a failure instead.</p>"
```